### DISCUSSION TITLE
Can Prisma Schemas also be used on the front-end?

### DISCUSSION THREAD

**Discussion by MentalGear** [prisma/prisma#13443] [Q&A]
I would like to unify my front and backend validation.
Therefore, it would be great if I could use the Prisma Schema on the front-end as well to validate form input on change before submit to the backend. Is this currently possible?

**Comment by nurul3101** [comment]
Hey @MentalGear 👋 

You could create a private npm package which exports the types generated by PrismaClient based on your schema and you could use this npm package in your frontend repository for validations.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
